csv/read.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/cloudquery/plugin-sdk/schema"
 )
 
-func Read(f io.Reader, table *schema.Table, sourceName string, res chan<- []any) error {
-	reader := csv.NewReader(f)
+func (*Client) Read(r io.Reader, table *schema.Table, res chan<- []any) error {
```

Comment:
why remove `sourceName` though?

Reply:
Because it wasn't being used... This comes back to [this comment here](https://github.com/cloudquery/filetypes/pull/30#discussion_r1070270463)

Reply:
I can leave it if it is part of a consistent function signature

Reply:
It is used in the testing. I think we should leave it.

Reply:
Done